src/sparseml/pytorch/sparsification/pruning/modifier_pruning_base.py

```diff
@@ -295,13 +299,15 @@ def initialize(
         layer_names = [nlp.layer_name for nlp in named_layers_and_params]
```

Comment:
nit: nlp stands for named layers and params right? a bit misleading, is it conventional naming?

Reply:
not great naming agreed, going to call this out of scope for this PR though esp since it's just internal to a list comp